### ISSUE TITLE
关于两种实现中前向函数的差别

### ISSUE THREAD

**Issue by rrjia** [wangx404/Center_Loss_in_MXNet#1]
您好，shownx实现的centerloss中 def hybrid_forward(self, F, feature, label, centers):有一个参数centers，但是我观察他的main.py代码时发现并没有传入这个参数，
if args.center_loss:
                    loss_center = center_loss(features, label)
                    loss = loss_softmax + loss_center
请问这个是否是一个错误？
